lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -840,12 +840,16 @@ def cmd_reg(*args)
           end
 
           v = open_key.query_value(value)
+          data = v.data
+          if v.type == REG_BINARY
+            data = data.unpack('H*')[0]
```

Comment:
Maybe use Rex::Text.to_hex() here?

I'm personally a fan of Rex::Text.to_hex_dump()

Reply:
On second thought, the output here is still more readable in context than the ones I mentioned, never mind.